tests/skr/key.py

```diff
@@ -66,10 +68,10 @@ def deploy_key(
         data=json.dumps(
             {
                 "key": {
-                    "kty": "oct-HSM",
+                    "kty": kty,
```

Comment:
does `kty` need a default value since it's replacing a hardcoded string? It would be nice for a comment to list out the possible values for kty and key__ops as well